src/main/java/seedu/address/model/person/Uid.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.person;
+
+import java.util.UUID;
+/**
+ * Uid that uniquely defines a person.
```

Comment:
I think it should be "identifies"

```suggestion
 * Uid that uniquely identifies a person.
```